src/TicketBai/Mews.Fiscalizations.TicketBai/ModelToDtoConverter.cs

```diff
@@ -0,0 +1,395 @@
+                DescripcionDetalle = item.Description.Value,
+                Cantidad = item.Quantity.ToString(),
+                ImporteUnitario = item.UnitAmount.ToString(),
+                Descuento = item.Discount.ToString(),
+                ImporteTotal = item.TotalAmount.ToString()
+            };
+        }
+
+        private static IDClaveType Convert(TaxMode taxMode)
+        {
+            return new IDClaveType
+            {
+                ClaveRegimenIvaOpTrascendencia = taxMode.Match(
+                    TaxMode.GeneralTaxRegimeActivity, _ => IdOperacionesTrascendenciaTributariaType.Item01,
+                    TaxMode.Export, _ => IdOperacionesTrascendenciaTributariaType.Item02,
+                    TaxMode.WorksOfArt, _ => IdOperacionesTrascendenciaTributariaType.Item03,
+                    TaxMode.InvestmentGold, _ => IdOperacionesTrascendenciaTributariaType.Item04,
+                    TaxMode.TravelAgencies, _ => IdOperacionesTrascendenciaTributariaType.Item05,
+                    TaxMode.GroupsOfEntities, _ => IdOperacionesTrascendenciaTributariaType.Item06,
+                    TaxMode.CashBasis, _ => IdOperacionesTrascendenciaTributariaType.Item07,
+                    TaxMode.CanaryIslandsGeneralIndirectTax, _ => IdOperacionesTrascendenciaTributariaType.Item08,
+                    TaxMode.TravelAgencyServicesActingAsIntermediaries, _ => IdOperacionesTrascendenciaTributariaType.Item09,
+                    TaxMode.Collections, _ => IdOperacionesTrascendenciaTributariaType.Item10,
+                    TaxMode.BusinessPremisesLeaseActivities1, _ => IdOperacionesTrascendenciaTributariaType.Item11,
+                    TaxMode.BusinessPremisesLeaseActivities2, _ => IdOperacionesTrascendenciaTributariaType.Item12,
+                    TaxMode.BusinessPremisesLeaseActivities3, _ => IdOperacionesTrascendenciaTributariaType.Item13,
+                    TaxMode.InvoiceWithVATPendingAccrual1, _ => IdOperacionesTrascendenciaTributariaType.Item14,
+                    TaxMode.InvoiceWithVATPendingAccrual2, _ => IdOperacionesTrascendenciaTributariaType.Item15,
+                    TaxMode.SurchargeOperations, _ => IdOperacionesTrascendenciaTributariaType.Item51,
+                    TaxMode.SimplifiedOperations, _ => IdOperacionesTrascendenciaTributariaType.Item52,
+                    TaxMode.NotVatEntity, _ => IdOperacionesTrascendenciaTributariaType.Item53
+                )
+            };
+        }
+
+        private static CabeceraFacturaType1 Convert(InvoiceHeader header)
+        {
+            return new CabeceraFacturaType1
+            {
+                SerieFactura = header.Series.Map(s => s.Value).GetOrNull(),
+                NumFactura = header.Number.Value,
+                FechaExpedicionFactura = Convert(header.IssueDate),
+                HoraExpedicionFactura = header.IssueDateTime.ToString("HH:MM:ss"),
+                FacturaSimplificada = header.IsSimplified.Match(
+                    t => SiNoType.S,
```

Comment:
I have to say that programming syntax of `YesNoClass.Y` is the best way to send boolean across network 😂